content/articles/how-to-create-font-chooser-app/index.md

```diff
@@ -0,0 +1,139 @@
+
+### How to create a font chooser app using python 
+A font picker is a simple application that allows you to browse through all the fonts installed on your computer, and filter down your options to find the ideal one for your project.
+
+The Tkinter Python library will be used to construct a font chooser application in this tutorial.
+
+### Table of content.
+- Prerequisite
+- Functions of tkinter
+- Graphical User Interface
+- Creating a font chooser app
+- Conclusion
+- Reference
+
+### Prerequisite
+To follow along with this tutorial, you will need to have:
+
+- The latest version of [Python3](https://www.python.org/downloads/) installed on your computer system.
+- A code editor, i use [Pycharm](https://www.jetbrains.com/pycharm/download/).
+
+### Functions of Tkinter
+Python's standard GUI library is Tkinter. When Python and Tkinter are integrated, creating graphical user interfaces becomes much faster and easier. 
+
+Tkinter is a useful data analysis tool. Data analysis is the systematic application of statistical tools to explain, show, condense and appraise data.
+
+Tkinter provides geometric widget configuration, which organizes widgets in parent windows for data science.
+
+There are three geometry manager classes in Tkinter. These are:
+
+- `.pack()` : It's used to organize widgets in blocks before they're placed in parent widgets.
+- `.grid()` : Before inserting widgets in the parent widget, grid() is used to organize them into a table-like layout.
+- `.place()` : This function is used to organize widgets by placing them in precise positions as specified by the programmer.
+
+Tkinter can be used to construct a variety of charts (pie charts, bar charts, and line charts), which are essential for data analysis.
+
+Tkinter's Combobox can be used to plot data points in charts. You might use `matplotlib` to create dynamic values that change the diagram depending on the values selected in the Combobox.
+
+Tkinter is used in creating GUI applications.
+ 
+You can install Tkinter by using the `pip` command.
+
+```python
+pip install tk
+```
+
+### The graphical user interface
+The graphical user interface is the most common type of user interfaces. One can interact with these interfaces by pointing and clicking on graphics or icons with a mouse, trackpad, or other peripherals.
+
+#### Advantages of the graphical user interface
+- It is simple to use.
```

Comment:
1st statement in this list:

The GUI is not simple to use. It's simpler than the CLI. Please update your statement to reflect this.